ClubManagement/ClubManagement/Fragments/EventDialogFragment.cs

```diff
@@ -73,7 +73,7 @@ private void OnClickSave(object sender, EventArgs e)
                     Longitude = place.LatLng.Longitude,
                     Latitude = place.LatLng.Latitude,
                     Place = place.AddressFormatted.ToString(),
-                    Time = date,
+                    TimeStart = date,
```

Comment:
Is `TimeEnd` missing here?

Reply:
this file is unused and will be removed in next version

Reply:
Done. This file was deleted